db/migration/V020__insert_default_activities.sql

```diff
@@ -0,0 +1,125 @@
+INSERT INTO curriculums (principal_id, title) VALUES
+  (1, 'Professional Mentorship Program');
+
+INSERT INTO activities (title, description_text, curriculum_week, curriculum_day, start_time, end_time, duration, activity_type_id, curriculum_id) VALUES
+  -- Week 1: Day 1
+  ('Morning Standup', '', 1, 1, '10:00:00', '11:00:00', 60, 3, 1),
+  ('Self-introduction', 'Get to know each other.', 1, 1, '11:10:00', '12:00:00', 50, 1, 1),
+  -- Week 1: Day 2
+  ('Self-assessment', '', 1, 2, NULL, NULL, NULL, 2, 1),
+  ('PMP Inroduction', 'Overview of the project and technological environment.', 1, 2, '11:00:00', '12:00:00', 60, 1, 1),
```

Comment:
"Introduction" is misspelled.

Reply:
Fix it in the commit: https://github.com/OpenTree-Education/rhizone-lms/pull/421/commits/3a224d805fedcf773be3ee52afb57ebe75381287